### PR TITLE
fix: oas2tson throwing error when path -p option is provided

### PR DESCRIPTION
**Pull request by faizplus**
when running ` "oas2tson": "./dist/esm/src/cli.js oas2tson -i ./example/openapi.yml -o ./example/output/ts -p paths",` throws error as invalid constants are created from file names for paths for example for path `pet/findByStatus` it creates 

```javascript
const path-findByStatus = {...} // invalid variable name
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,13 +40,17 @@
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^11.1.0",
     "@openapi-contrib/openapi-schema-to-json-schema": "^5.1.0",
+    "@types/lodash.camelcase": "^4.3.9",
```
Comment (ilteoood) [inline package.json:43]: types should stay in the `devDependencies`

**Review by ilteoood — CHANGES_REQUESTED** [review]

**Comment by ilteoood** [comment]
Additionally, it's strange that no tests are breaking. Can you add a test to cover this?

File: package.json (hunk 1 above)
Comment (faizplus) [reply package.json:43]: right, missed it

**Comment by faizplus** [comment]
Yes, there is no test to cover this, let me add one

**Review by ilteoood — APPROVED** [review]
LGTM

**Review by simoneb — APPROVED** [review]

**Comment by jmjf** [comment]
This change to filename generation causes `oas2tson` to fail with schema names like `FooBARBaz`. See https://github.com/nearform/openapi-transformer-toolkit/issues/294 for details.

**Comment by simoneb** [comment]
@jmjf would you be interested in submitting a PR?

**Comment by jmjf** [comment]
I'll work on a PR to revert this PR, yes.

**Pull request merged**